src/assets/fonts/material-icons.css

```diff
@@ -16,6 +16,7 @@
   font-family: "Material Icons";
   font-style: normal;
   font-weight: 400;
+  font-display: swap;
```

Comment:
I think font-display: swap might not be the right option for icon fonts (the fallback font would be of no use to users)

Reply:
Oh, that is actually a good argument, and something Google does not consider in their lighthouse checks.

Reply:
As a general note: we shouldn't optimize for 100% lighthouse score. Increasing the score by providing better accessibility or faster load time is great, but some things make no sense: https://www.matuzo.at/blog/building-the-most-inaccessible-site-possible-with-a-perfect-lighthouse-score/

Reply:
Ref: 
- https://github.com/FortAwesome/Font-Awesome/issues/14387
- https://www.zachleat.com/web/font-display-icon-fonts/
- https://stackoverflow.com/questions/49461308/correct-font-display-value-for-icon-fonts

Reply:
I filed https://github.com/GoogleChrome/lighthouse/issues/10127

Reply:
Already got an answer, https://github.com/GoogleChrome/lighthouse/issues/10127#issuecomment-567093362

So it is not actually looking for `swap` but for any flag, the maintainer also commented on the FontAwesome issue about this, informing them of that.

So two choices: 
- We set it to block for our icons fonts
- We don't change it and hope that FA and MaterialIcons adopt the change. I would create an issue on the MI repo as well.